CMakeLists.txt

```diff
@@ -145,8 +145,11 @@ if(WIN32)
     add_definitions(-DBOOST_INTERPROCESS_BOOTSTAMP_IS_SESSION_MANAGER_BASED)
 endif()
 
-add_definitions(-DBOOST_CHRONO_HEADER_ONLY -DBOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE)
+add_definitions(-DBOOST_CHRONO_HEADER_ONLY -DBOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE -DBOOST_FILESYSTEM_NO_CXX20_ATOMIC_REF)
```

Comment:
Do you have any idea why we have to set this manually? It seems that it should be [detected automatically](https://github.com/boostorg/filesystem/blob/boost-1.80.0/CMakeLists.txt#L35), but then I don't see `has_cxx20_atomic_ref.cpp` in the extracted boost source...

Reply:
I suspect bcp is not smart enough to include has_cxx20_atomic_ref.cpp because it is only referenced in the CMakeLists.txt and not in the actual source code?

Reply:
Not sure if that's relevant, but a shot in the dark - did you use bcp from boost 1.80?

Reply:
I did indeed. I didn't find any package for bcp so had to build it myself.

Reply:
@dyfer Are any more actions expected from me?

Reply:
@shimpe thanks for checking - nothing at this time, thanks. 
We might want to pull it into 3.13.1 (not sure yet) so it's a matter of timing this right.